### PR TITLE
Docs home page

### PR DESCRIPTION
**Pull request by bmbshlly**

### CODE REVIEW

File: components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+            css={{
+                borderTopWidth: '1px',
+                borderTopStyle: 'solid',
+                borderTopColor: '$surfaceLight',
+                paddingTop: '$14'
+            }}>
+            <Flex align="center" direction="column" className="footer-wrapper">
+                <Flex className="footer" justify="between">
+                    <Box>
+                        <img height="36" src="/docs/100ms.svg" alt="100ms Logo" />
```
Comment (bmbshlly) [inline components/Footer.tsx:25]: use react-ui library for icons @KaustubhKumar05

File: components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+                                            <Text css={{ color: '$textHighEmp' }}>
+                                                Subscribe for developer updates
+                                            </Text>
+                                            <Input
+                                                className="footer-input"
+                                                type="email"
+                                                placeholder="Enter your email"
+                                            />
+                                            <Text variant="sm" css={{ color: '$textMedEmp' }}>
+                                                You can unsubscribe anytime
```
Comment (bmbshlly) [inline components/Footer.tsx:126]: remove unused code @KaustubhKumar05

File: components/Card.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { Box, Flex, Text, useTheme } from '@100mslive/react-ui';
+import React from 'react';
+
+interface Props {
+    body: any;
+    css?: {};
```
Comment (hardik1003) [inline components/Card.tsx:7]: should import CSS from the react-ui interface

File: components/Card.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    endComponent?: any;
```
Comment (hardik1003) [inline components/Card.tsx:8]: Should be React.FC

File: components/Card.tsx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    titleComponent?: any;
```
Comment (hardik1003) [inline components/Card.tsx:9]: Same

File: components/DropDownOption.tsx [hunk 6]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable react/react-in-jsx-scope */
+import { Flex, Text } from '@100mslive/react-ui';
+
+const DropDownOption = ({ icon, title, setDropDownSelection }) => (
```
Comment (hardik1003) [inline components/DropDownOption.tsx:4]:
```
const DropDownOption = ({ icon, title, setDropDownSelection }) =
```
```suggestion
const getButtonVariants = (base: string, hover: string, active: string, disabled: string) => {
  
};
OR 
type Props = VariantProps<typeof StyledAvatar> &
  React.ComponentProps<typeof StyledAvatar> & {
    name: string;
  };

export const Avatar: React.FC<Props>
```

File: components/Footer.tsx [hunk 7]
```diff
@@ -0,0 +1,214 @@
...
+    DiscordIcon,
+    FacebookIcon,
+    LinkedInIcon,
+    Svg100MsLogo,
+    TwitterIcon
+} from '@100mslive/react-icons';
+import { Box, Flex, HorizontalDivider, Text } from '@100mslive/react-ui';
+import { useStatusPage } from 'hooks';
+
+const Footer = ({ css = {} }) => (
```
Comment (hardik1003) [inline components/Footer.tsx:14]: add type

File: components/Header.tsx [hunk 8]
```diff
@@ -1,12 +1,16 @@
-import SearchIcon from '@/assets/icons/SearchIcon';
-import SvgMoon from '@/assets/icons/Moon';
-import SvgSun from '@/assets/icons/Sun';
-import CrossIcon from '@/assets/icons/CrossIcon';
-import MenuIcon from '@/assets/icons/MenuIcon';
-import React from 'react';
+import { useTheme } from '@100mslive/react-ui';
+import {
```
Comment (hardik1003) [inline components/Header.tsx:2]: should come before

File: components/HomeDropDown.tsx [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+    AppleIcon,
+    FlutterIcon,
+    JavascriptIcon,
+    ReactIcon,
+    ServerIcon
+} from '@100mslive/react-icons';
+import { Box, Flex, Text } from '@100mslive/react-ui';
+import DropDownOption from './DropDownOption';
+
+const HomeDropDown = ({ setDropDownSelection }) => {
```
Comment (hardik1003) [inline components/HomeDropDown.tsx:13]: where is type?

File: components/Item.tsx [hunk 10]
```diff
@@ -0,0 +1,51 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { ChevronRightIcon } from '@100mslive/react-icons';
+import { Flex, Text } from '@100mslive/react-ui';
+import React from 'react';
+
+interface Props {
+    logo?: React.SVGProps<SVGSVGElement>;
+    endLogo?: React.SVGProps<SVGSVGElement> | null;
+    css?: {};
+    textCSS?: {};
```
Comment (hardik1003) [inline components/Item.tsx:10]: ???

File: components/Item.tsx [hunk 11]
```diff
@@ -0,0 +1,51 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { ChevronRightIcon } from '@100mslive/react-icons';
+import { Flex, Text } from '@100mslive/react-ui';
+import React from 'react';
+
+interface Props {
+    logo?: React.SVGProps<SVGSVGElement>;
+    endLogo?: React.SVGProps<SVGSVGElement> | null;
+    css?: {};
```
Comment (hardik1003) [inline components/Item.tsx:9]: ???

File: components/Item.tsx [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+    textCSS?: {};
+    text: string;
+    textVariant?: any;
```
Comment (hardik1003) [inline components/Item.tsx:12]: should be string

File: components/SdkItem.tsx [hunk 13]
```diff
@@ -0,0 +1,256 @@
...
+                                }}
+                            />
+                        </Box>
+                    </a>
+                </Link>
+            ))}
+        </Box>
+    </Flex>
+
+    // <Flex
```
Comment (hardik1003) [inline components/SdkItem.tsx:211]: comment??
please delete

File: components/SearchModal.tsx [hunk 14]
```diff
@@ -1,14 +1,14 @@
...
-import Link from 'next/link';
 import useKeyPress from '@/lib/useKeyPress';
+import useSearch from '@/lib/useSearch';
+import Link from 'next/link';
+import React from 'react';
 
 interface Props {
-    docs: { url: string; title: string; description: string; nav: number; content: string }[];
-    currentDocSlug: string;
+    docs?: { url: string; title: string; description: string; nav: number; content: string }[];
```
Comment (hardik1003) [inline components/SearchModal.tsx:10]: why '?' ?

File: components/Switcher.tsx [hunk 15]
```diff
@@ -0,0 +1,40 @@
...
+
+interface TabObject {
+    id: number;
+    name: string;
+}
+
+interface Props {
+    tabs: TabObject[];
+    activeTab: string;
+    setActiveTab: any;
```
Comment (hardik1003) [inline components/Switcher.tsx:14]: why is there any?

**Review by hardik1003 — APPROVED** [review]

File: components/Card.tsx [hunk 16]
```diff
@@ -0,0 +1,53 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { Box, Flex, Text, useTheme } from '@100mslive/react-ui';
+import React from 'react';
+
+interface Props {
+    body: any;
```
Comment (hardik1003) [inline components/Card.tsx:6]: React.FC

**Review by hardik1003 — CHANGES_REQUESTED** [review]

File: components/DropDownOption.tsx (hunk 6 above)
Comment (hardik1003) [inline components/DropDownOption.tsx:4]: please add type

File: components/Footer.tsx [hunk 17]
```diff
@@ -0,0 +1,147 @@
...
+                                            color: 'inherit'
+                                        }}
+                                    />
+                                </Box>
+                            </a>
+                            <a href="https://www.facebook.com/100mslive">
+                                <Box css={{ color: '$textMedEmp' }}>
+                                    <FacebookIcon
+                                        style={{
+                                            width: '36px',
```
Comment (hardik1003) [inline components/Footer.tsx:49]: please enter rem values

File: components/Footer.tsx [hunk 18]
```diff
@@ -0,0 +1,147 @@
...
+                                            color: 'inherit'
+                                        }}
+                                    />
+                                </Box>
+                            </a>
+                            <a href="https://twitter.com/100mslive">
+                                <Box css={{ color: '$textMedEmp' }}>
+                                    <TwitterIcon
+                                        style={{
+                                            width: '27px',
```
Comment (hardik1003) [inline components/Footer.tsx:60]: same

File: components/Footer.tsx [hunk 19]
```diff
@@ -0,0 +1,147 @@
...
+                                            top: '5px'
+                                        }}
+                                    />
+                                </Box>
+                            </a>
+                            <a href="https://discord.com/invite/kGdmszyzq2">
+                                <Box css={{ color: '$textMedEmp' }}>
+                                    <DiscordIcon
+                                        style={{
+                                            width: '36px',
```
Comment (hardik1003) [inline components/Footer.tsx:73]: same

**Review by hardik1003 — APPROVED** [review]

File: components/Card.tsx [hunk 20]
```diff
@@ -0,0 +1,53 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```
Comment (hardik1003) [inline components/Card.tsx:1]: ?

File: components/DropDownOption.tsx [hunk 21]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable react/react-in-jsx-scope */
```
Comment (hardik1003) [inline components/DropDownOption.tsx:1]: import React, { Component }  from 'react';

File: components/Footer.tsx [hunk 22]
```diff
@@ -0,0 +1,147 @@
+/* eslint-disable react/no-deprecated */
+/* eslint-disable react/react-in-jsx-scope */
```
Comment (hardik1003) [inline components/Footer.tsx:2]: please remove this

File: components/Footer.tsx [hunk 23]
```diff
@@ -0,0 +1,147 @@
+/* eslint-disable react/no-deprecated */
```
Comment (hardik1003) [inline components/Footer.tsx:1]: why this

File: components/Footer.tsx [hunk 24]
```diff
@@ -0,0 +1,147 @@
...
+            justify="center"
+            css={{
+                borderTopWidth: '1px',
+                borderTopStyle: 'solid',
+                borderTopColor: '$borderDefault',
+                paddingTop: '$14'
+            }}>
+            <Flex align="center" direction="column" className="footer-wrapper">
+                <Flex className="footer" justify="between">
+                    <Box>
```
Comment (hardik1003) [inline components/Footer.tsx:31]: extra div

File: components/HomeDropDown.tsx [hunk 25]
```diff
@@ -0,0 +1,102 @@
+/* eslint-disable react/react-in-jsx-scope */
```
Comment (hardik1003) [inline components/HomeDropDown.tsx:1]: React import should be at top and remove this

File: components/Card.tsx [hunk 26]
```diff
@@ -0,0 +1,53 @@
...
+import { Box, CSS, Flex, Text, useTheme } from '@100mslive/react-ui';
+import React from 'react';
```
Comment (hardik1003) [inline components/Card.tsx:3]: send this to top

File: components/Footer.tsx [hunk 27]
```diff
@@ -0,0 +1,147 @@
...
+    ChatIcon,
+    DiscordIcon,
+    FacebookIcon,
+    LinkedInIcon,
+    Svg100MsLogo,
+    TwitterIcon
+} from '@100mslive/react-icons';
+import { Box, CSS, Flex, HorizontalDivider, Text } from '@100mslive/react-ui';
+import { useStatusPage } from 'hooks';
+import React from 'react';
```
Comment (hardik1003) [inline components/Footer.tsx:13]: Import react at top

File: components/HomeDropDown.tsx [hunk 28]
```diff
@@ -0,0 +1,102 @@
...
+import {
+    AndroidIcon,
+    AppleIcon,
+    FlutterIcon,
+    JavascriptIcon,
+    ReactIcon,
+    ServerIcon
+} from '@100mslive/react-icons';
+import { Box, Flex, Text } from '@100mslive/react-ui';
+import React from 'react';
```
Comment (hardik1003) [inline components/HomeDropDown.tsx:11]: Should import at top

File: components/Item.tsx [hunk 29]
```diff
@@ -0,0 +1,51 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```
Comment (hardik1003) [inline components/Item.tsx:1]: why?

File: components/Item.tsx [hunk 30]
```diff
@@ -0,0 +1,51 @@
...
+import { CSS, Flex, Text } from '@100mslive/react-ui';
+import React from 'react';
+
+interface Props {
+    logo?: React.SVGProps<SVGSVGElement>;
+    endLogo?: React.SVGProps<SVGSVGElement> | null;
+    css?: CSS;
+    textCSS?: CSS;
+    text: string;
+    textVariant?: any;
```
Comment (hardik1003) [inline components/Item.tsx:12]: should be string

File: components/SdkItem.tsx [hunk 31]
```diff
@@ -0,0 +1,211 @@
...
+    ListCheckIcon,
+    RestApiIcon,
+    RocketIcon,
+    WebhookIcon
+} from '@100mslive/react-icons';
+import { Box, CSS, Flex, HorizontalDivider, VerticalDivider } from '@100mslive/react-ui';
+import { Item } from 'components';
+import merge from 'lodash/merge';
+import Link from 'next/link';
+import React from 'react';
```
Comment (hardik1003) [inline components/SdkItem.tsx:16]: should go to top

File: components/SdkItem.tsx [hunk 32]
```diff
@@ -0,0 +1,211 @@
...
+    1: <BookIcon style={iconStyle} />,
+    2: <CodeIcon style={iconStyle} />,
+    3: <GithubIcon style={iconStyle} />,
+    4: <RestApiIcon style={iconStyle} />,
+    5: <RocketIcon style={iconStyle} />,
+    6: <WebhookIcon style={iconStyle} />,
+    7: <ListCheckIcon style={iconStyle} />
+};
+
+const sdkItems = {
```
Comment (hardik1003) [inline components/SdkItem.tsx:40]: move to down

File: components/SdkItem.tsx [hunk 33]
```diff
@@ -0,0 +1,211 @@
...
+    3: 'Sample app',
+    4: 'API reference',
+    5: 'Release notes',
+    6: 'Webhook',
+    7: 'Domain and port whitelisting'
+};
+
+const iconStyle = { height: '20px', color: '$textMedEmp' };
+
+const itemList = {
```
Comment (hardik1003) [inline components/SdkItem.tsx:30]: move to bottom

File: components/SdkItem.tsx [hunk 34]
```diff
@@ -0,0 +1,211 @@
...
+    RocketIcon,
+    WebhookIcon
+} from '@100mslive/react-icons';
+import { Box, CSS, Flex, HorizontalDivider, VerticalDivider } from '@100mslive/react-ui';
+import { Item } from 'components';
+import merge from 'lodash/merge';
+import Link from 'next/link';
+import React from 'react';
+
+const itemId = {
```
Comment (hardik1003) [inline components/SdkItem.tsx:18]: move to bottom

File: components/SearchModal.tsx [hunk 35]
```diff
@@ -1,14 +1,13 @@
...
-import SearchIcon from '@/assets/icons/SearchIcon';
-import useSearch from '@/lib/useSearch';
 import EnterIcon from '@/assets/icons/EnterIcon';
+import SearchIcon from '@/assets/icons/SearchIcon';
 import useClickOutside from '@/lib/useClickOutside';
-import Link from 'next/link';
 import useKeyPress from '@/lib/useKeyPress';
+import useSearch from '@/lib/useSearch';
+import Link from 'next/link';
+import React from 'react';
```
Comment (hardik1003) [inline components/SearchModal.tsx:7]: ????

File: components/Switcher.tsx [hunk 36]
```diff
@@ -0,0 +1,40 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```
Comment (hardik1003) [inline components/Switcher.tsx:1]: ??

File: components/Switcher.tsx [hunk 37]
```diff
@@ -0,0 +1,40 @@
...
+import { Flex } from '@100mslive/react-ui';
```
Comment (hardik1003) [inline components/Switcher.tsx:2]: Please import react first

File: components/SwitcherTab.tsx [hunk 38]
```diff
@@ -0,0 +1,36 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```
Comment (hardik1003) [inline components/SwitcherTab.tsx:1]: ?

File: pages/[...slug].tsx [hunk 39]
```diff
@@ -1,19 +1,19 @@
 /* eslint-disable global-require */
```
Comment (hardik1003) [inline pages/[...slug].tsx:1]: ?

File: pages/_app.tsx [hunk 40]
```diff
@@ -1,19 +1,27 @@
 /* eslint-disable import/no-extraneous-dependencies */
```
Comment (hardik1003) [inline pages/_app.tsx:1]: ?

File: pages/_app.tsx [hunk 41]
```diff
@@ -1,19 +1,27 @@
...
-import React from 'react';
+import '@/styles/theme.css';
+import 'inter-ui/inter.css';
 import { NextPage } from 'next';
-import { AppProps } from 'next/app';
 import { DefaultSeo } from 'next-seo';
+import { AppProps } from 'next/app';
+import dynamic from 'next/dynamic';
 import { useRouter } from 'next/router';
+import React from 'react';
```
Comment (hardik1003) [inline pages/_app.tsx:9]: React should be at top

File: pages/[...slug].tsx (hunk 39 above)
Comment (hardik1003) [reply pages/[...slug].tsx:1]: https://eslint.org/docs/latest/rules/global-require

File: components/SdkItem.tsx [hunk 42]
```diff
@@ -0,0 +1,211 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```
Comment (hardik1003) [inline components/SdkItem.tsx:1]: ?

File: components/SdkItem.tsx [hunk 43]
```diff
@@ -0,0 +1,211 @@
...
+        { id: 1, link: '/react-native/v2/guides/quickstart' },
+        { id: 3, link: 'https://github.com/100mslive/react-native-hms' },
+        { id: 4, link: '/api-reference/react-native/v2/modules.html' },
+        { id: 5, link: '/react-native/v2/release-notes/release-notes' }
+    ]
+};
+
+interface Props {
+    logo?: React.SVGProps<SVGSVGElement>;
+    sdk?: any;
```
Comment (hardik1003) [inline components/SdkItem.tsx:90]: should be string

File: components/DropDownOption.tsx (hunk 21 above)
Comment (hardik1003) [reply components/DropDownOption.tsx:1]: https://stackoverflow.com/questions/42640636/react-must-be-in-scope-when-using-jsx-react-react-in-jsx-scope

File: components/Footer.tsx (hunk 23 above)
Comment (hardik1003) [reply components/Footer.tsx:1]: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-deprecated.md

File: components/Footer.tsx (hunk 22 above)
Comment (hardik1003) [reply components/Footer.tsx:2]: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-deprecated.md

**Review by hardik1003 — CHANGES_REQUESTED** [review]

**Comment by hardik1003** [comment]
Resolve conflict

File: components/SdkItem.tsx (hunk 43 above)
Comment (bmbshlly) [reply components/SdkItem.tsx]: image

File: components/SdkItem.tsx [hunk 44]
```diff
@@ -109,10 +109,19 @@ const SdkItem: React.FC<Props> = ({ logo, text, sdk, css, cssHeading }) => (
...
             }}>
             {sdkItems[sdk as keyof typeof sdkItems].map((value) => (
-                <Link key={value.id} href={value.link}>
+                <Link key={value.id} href={value.link} passHref>
                     <a style={{ width: 'fit-content' }}>
                         <Box
                             className="hoverParent"
+                            role="link"
+                            onClick={() =>
+                                window.analytics.track('onClick', {
```
Comment (hardik1003) [inline components/SdkItem.tsx]: track event shoould be fixed

**Review by hardik1003 — APPROVED** [review]

File: components/Card.tsx [hunk 45]
```diff
@@ -6,7 +6,8 @@ interface Props {
     css?: CSS;
     endComponent?: React.ReactNode;
     titleComponent?: React.ReactNode;
-    onClick?: Function;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (hardik1003) [inline components/Card.tsx:9]: ? use ()=>void

**Pull request merged**